web/styles/inbox.css

```diff
@@ -330,13 +331,55 @@
             }
         }
 
+        .inbox-right-part-wrapper {
+            display: flex;
+            align-items: center;
+        }
+
         #inbox_filter_mute_toggle {
             font-size: 16px;
             width: 16px;
             height: 16px;
             position: relative;
             top: 1px;
         }
+
+        .inbox-row {
+            &:focus,
+            &:focus-within,
+            &:hover {
+                .inbox-action-button {
+                    opacity: 1;
+                }
+            }
+        }
+
+        .inbox-action-button {
+            display: flex;
+            border-radius: 3px;
+            outline: none;
+            opacity: 0;
+
+            &.hide {
+                display: none;
+            }
+
+            &:focus {
+                box-shadow: 0 0 0 2px var(--color-outline-focus);
+            }
+
+            & i {
+                padding: 5px;
+                font-size: 16px;
+                opacity: 0.3;
+                color: var(--color-vdots-hover);
+
+                &:hover {
+                    opacity: 1;
+                    cursor: pointer;
+                }
+            }
+        }
```

Comment:
What's the thinking on the color choices? It looks like the on-row-hover (but no on-element-hover) state is a lot less bright than the left sidebar variant:

![image](https://github.com/zulip/zulip/assets/2746074/2b7ebf59-d302-48cb-bf29-c97629903256)

@karlstolley FYI in case you have suggestions for how we should fix this.

(In any case we can do in a follow-up PR, since the feature isn't launched yet)

Reply:
It's based on figma design. https://www.figma.com/file/jbNOiBWvbtLuHaiTj4CW0G/Zulip-Web-App?node-id=2189%3A180439&mode=dev
<img width="582" alt="Screenshot 2023-09-22 at 1 28 47 PM" src="https://github.com/zulip/zulip/assets/25124304/7910ba3e-2114-4ffd-bad1-e1655c2e63e1">

Reply:
Hmmmm. I wouldn't use the hover-state color when they're just visible; is `--color-vdots-visible` insufficient with the background in the inbox?

Also, I'd like to get away from separate `opacity` properties that are intended for colors; those should be expressed right with the color on the `hsl()` syntax (and IIRC, `--color-vdots-hover` is at 100% opacity anyway.